CRM/Contact/BAO/Query.php

```diff
@@ -6355,15 +6355,14 @@ public static function processSpecialFormValue(&$formValues, $specialFields, $ch
    * @param string|CRM_Utils_Sort $sort
    *   The order by string.
    * @param null $sortOrder
-   *   Who knows? Hu knows. He who knows Hu knows who.
+   *   Who knows? Hu knows. One who knows Hu knows who.
    *
    * @return string
    *   list(string $orderByClause, string $additionalFromClause).
    *
    * @throws \CRM_Core_Exception
    */
   protected function prepareOrderBy($sort, $sortOrder) {
-    $orderByArray = [];
```

Comment:
Was this seemingly unrelated change intended as part of this commit?

Reply:
Yes, see **Technical Details** of the description :)

Reply:
Ha, I forgot to read. :) Oops.